templates/js/copilot-plugin-from-scratch/.vscode/launch.json

```diff
@@ -56,19 +62,13 @@
                 "hidden": true
             },
             "internalConsoleOptions": "neverOpen"
-        },
-        {
-            "name": "Attach to Node Functions",
-            "type": "node",
-            "request": "attach",
-            "port": 9229,
-            "preLaunchTask": "func: host start"
         }
     ],
     "compounds": [
         {
-            "name": "Debug (Edge)",
+            "name": "Debug in Copilot (Edge)",
```

Comment:
@MuyangAmigo Do we need the original config of debugging this message extension?